### ISSUE TITLE
Can't Add Any Extras When Creating the GitHub Project

### ISSUE THREAD

**Issue by alexfinnarn** [mathieudutour/medium-to-own-blog#55]
**Describe the bug**

GitHub provides a few options when configuring a new repo like: license, .gitignore file, a README.md, and the Rennovate plugin. If you want to install any of these by default, which I usually do, then you get an error about the remote having newer content.

**To Reproduce**
1. Create a repo with a readme during the script
2. See the error.

**Screenshots**

<img width="707" alt="Screen Shot 2019-06-23 at 8 57 04 PM" src="https://user-images.githubusercontent.com/3640707/59984553-7b53de00-95f9-11e9-8a7c-99d55e67c82f.png">

<img width="775" alt="Screen Shot 2019-06-23 at 7 48 38 PM" src="https://user-images.githubusercontent.com/3640707/59984560-8575dc80-95f9-11e9-8693-c147d192d574.png">

**Comment by alexfinnarn** [comment]
https://github.com/mathieudutour/medium-to-own-blog/blob/master/bin/medium-to-own-blog.js#L148  

Why not `git clone ${remoteURL} && git add . && git commit -m "first commit :tada:" since the repo must exist at that point in the script?

**Comment by mathieudutour** [comment]
`git clone` is going to create a subfolder. I could `git pull` but there will be conflicts anyway. I kind of assume that if you use a repo with some content inside, the safest thing to do is to fail. There is no telling what kind of weird stuff might happen if we try to automatically fix the conflicts or force push

**Comment by alexfinnarn** [comment]
Makes sense. Do you think it is a good idea to add a note to the documentation somewhere? I can make a PR if you wish or close the issue if not. The gif/video creates one without any additional configuration, but a non-sighted user won't be able to see that. 

I think it would be most appropriate to add a line to https://github.com/mathieudutour/medium-to-own-blog/blob/master/bin/medium-to-own-blog.js#L94.

**Comment by mathieudutour** [comment]
yep, good idea! Happy to accept a PR to clarify that

**Comment by alexfinnarn** [comment]
Not sure of how to format this the best, but the language makes sense to me.
